bundlewrap/pyproject.toml

```diff
@@ -0,0 +1,16 @@
+[tool.poetry]
+name = "cm"
+version = "0.1.0"
+description = "Configuration management for all VOC systems"
+authors = ["c3voc"]
+
+[tool.poetry.dependencies]
```

Comment:
Is there any way to have this import the dependencies from `requirements.txt`? I'd like to avoid having to update dependencies in two places instead of one.

Reply:
Yes and no. You would have to 'poetry add <package>' from the requirements.txt.
But it will only install the latest version of the package.